{'project_id': '3130', 'year': '1984', 'x': 807420, 'y': 1974810, 'plot_id': '4566', 'strata': '41'}
Pandas(Index=0, project_id='3130', plot_id='4566', x=807420, y=1974810)
called process_plot
Pandas(Index=0, project_id='3130', plot_id='4566', x=807420, y=1974810)
4566
Directory '/efs/timesync/1984/audit/' already exists.
Data has been pickled and saved to '/efs/timesync/1984/audit/1984_4566_3130.p'.
............................Failed qa: 1984-07-04T15:51:16.5920940Z
Failed qa: 1984-07-11T15:57:30.9340750Z
.....................Failed qa: 1984-09-22T15:52:51.1460000Z
............................Data has been pickled and saved to '/efs/timesync/1984/audit/1984_4566_3130_qa_rejects.p'.
Success tracking goes here
